site/static/main.ts

```diff
@@ -72,7 +72,7 @@ function embedExample($target: any, spec: TopLevelExtendedSpec, actions=true, to
   });
 
   if (tooltip) {
-    vegaLite(view, spec);
```

Comment:
can you fix this in a separate PR that we can undo easily

Reply:
Yup